libmamba/src/api/list.cpp

```diff
@@ -209,9 +213,14 @@ namespace mamba
                                                   : compare_alphabetically;
                 std::sort(packages.begin(), packages.end(), comparator);
 
-                // format and print table
+                // format and print output
                 if (options.explicit_)
                 {
+                    if (options.canonical)
+                    {
+                        LOG_WARNING << "Warning: Option --canonical ignored because of --explicit"
```

Comment:
No need to add `Warning` in the message here as it's redundant with `LOG_WARNING` already mentioning it.
Same for `std::endl;`